### ISSUE TITLE
TOC stopped working

### ISSUE THREAD

**Issue by chilek** [vsch/idea-multimarkdown#751]
![Zrzut ekranu z 2019-05-17 11-14-22](https://user-images.githubusercontent.com/1457437/57917453-002e3800-7895-11e9-8f56-64b217213248.png)

Doesn't generate table of contents - only warning in tooltip as in screenshot. Tried to change contents a little bit, save document - it doesn't help :(

```
=== About ===
Build version: PhpStorm 2019.1.2 Build #PS-191.7141.52 May 8, 2019
Java version: 1.8.0_202-release-1483-b49amd64
Operating System: Linux (5.0.14-300.fc30.x86_64, amd64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o

=== System ===
Number of CPU: 8
Used memory: 771Mb 
Free memory: 391Mb 
Total memory: 1163Mb 
Maximum available memory: 1631Mb

=== Displays ===
Displays: 
Display 0: 2560x1440; scale: 1,00

=== Plugins ===
Custom plugins: [Markdown Navigator (2.8.4.28)]
Disabled plugins:[IntelliJ Configuration Script (191.7141.52), Markdown support (191.7141.52)]
```

**Comment by vsch** [comment]
@chilek, are you getting any exceptions?

The markdown code style setting sets whether toc is updated on save:

![image](https://user-images.githubusercontent.com/10299086/57942581-d3cbe900-789f-11e9-8d2c-67cc3c8d9691.png)

**Comment by chilek** [comment]
@vsch thanks for hint! Although you image is broken I was able to find this settings. Indeed - my almost all settings have been lost during struggle strange problems with global/project settings in my PHPStorm. I forgot that I had to reset some settings in my project. Now I adjusted settings for TOC and it works again. Nice side efect by the way: automatic scrolling in preview started working! ;-) Thanks for your help!
